### DISCUSSION TITLE
QQ plot interpretation

### DISCUSSION THREAD

**Discussion by AlineGiroux** [bmcclintock/momentuHMM#119] [Q&A] [answered]
Hi Brett, 
I'm using momentuHMM to model giant anteaters' GPS data. Although my sampling regime is pretty regular, I used the crawlWrap function to predict extremely temporally-regular tracks. 
When I use the resampled data, the residuals plot seems a little weird. The turn angles does not have a good fit, and I actually don't understand why the step length ACF switch between positive and negative values.  
![qq plot - resampled](https://user-images.githubusercontent.com/91481371/150847985-34cd9461-305c-4e81-80b5-47086586b1f4.png)
When I use the original data (not resampled with crawlWrap function) the residuals plot seems better. Although I'm still not sure if this decay in step length ACF values is good enough. 
![qq plot - original data](https://user-images.githubusercontent.com/91481371/150849964-ff8826de-db5b-4c3f-8a54-cea605842837.png)
In both cases (using and not using crawlWrap function), the model results are very similar.  
Can you please help me to understand why this divergence in residuals plot? 
In this case, do you advise me to use the original data (no resampling)?  

Thanks a lot

**Comment by bmcclintock** [comment, marked as answer]
I would need a lot more detail and a reproducible example to provide any advice on this one. There could be problems with the `crawlWrap` fits or other errors. Discrete-time HMMs require temporally-regular observations, so the original data may not be appropriate for this type of analysis unless "pretty regular" is _very_ close to regular.

Pseudo-residual plots can be tough to diagnose and require a very good understanding of the study system, data, and model being fit. For more on pseudo-residual plots, see #54 and #115 and tutorials such as [Zucchini et al. (2016)](https://doi.org/10.1201/b20790), [Pohle et al. (2017)](https://doi.org/10.1007/s13253-017-0283-8), [McClintock et al. (2020)](https://doi.org/10.1111/ele.13610), and the elephant example in the `momentuHMM` [vignette](https://cran.r-project.org/web/packages/momentuHMM/vignettes/momentuHMM.pdf).
